recipes/nyxus/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "nyxus" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PolusAI/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e82f349f6ff481e017c3ff2060642e66e81dbda5910272fba1b3c5a4b5d9b2fd
+  patches:
+    - 0001-fix-filesystem-issue.patch
+    - 0002-Fix-CUDA-Kernel-Issue.patch
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  missing_dso_whitelist:
+    - "*/nvcuda.dll"   # [win]
```

```suggestion
build:
  number: 0
  skip: true  # [py<36]
  missing_dso_whitelist:
    - "*/nvcuda.dll"   # [win]
  ignore_run_exports_from:
  # The CXX standard library is unused on linux, but used on osx
    - {{ compiler('cxx') }}  # [linux]
```
According to the link inspector the cxx standard library is unused on linux.